can.yml

```diff
@@ -0,0 +1,8 @@
+generator:
+  templateDirectory: ./templates/go-client
+  moduleName: github.com/sasswart/gin-in-a-can/cmd/test_api
+  basePackageName: api
+  InvalidRequestStatus: 499
```

Comment:
I changed this to 499 for debugging. Have since learned that this is a valid HTTP response code. 
Probably worth bringing it back to a suitable 4xx before merging

Reply:
`InvalidRequestStatus` is entirely defunct. Thanks for catching. I've purged it from the codebase.